test/unit/core.js

```diff
@@ -618,8 +618,12 @@ QUnit.test( "jQuery('html')", function( assert ) {
 
 	//equal( jQuery( "element[attribute=<div></div>]" ).length, 0,
 	//	"When html is within brackets, do not recognize as html." );
-	assert.equal( jQuery( "element:not(<div></div>)" ).length, 0,
-		"When html is within parens, do not recognize as html." );
+	if ( jQuery.find.compile ) {
+		assert.equal( jQuery( "element:not(<div></div>)" ).length, 0,
+			"When html is within parens, do not recognize as html." );
+	} else {
+		assert.ok( true, "Complex :not not supported in selector-native" );
```

Comment:
I like the idea of marking individual tests somehow, and "skip" seems as good as any. Seems like it would be a useful QUnit feature as well?